tests/test_optimize.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from unittest import TestCase
+
+from aspire.utils import ensure
+from aspire.utils.matlab_compat import m_reshape
+from aspire.utils.optimize import conj_grad
+
+
+def identity(x):
+    return x
+
+
+def precond_fun(S, x):
+    p = np.size(S, 0)
+    ensure(np.size(x) == p*p, 'The sizes of S and x are not consistent.')
+    x = m_reshape(x, (p, p))
```

Comment:
Why are we using `m_reshape` here? From what I understand, these functions were introduced to help transition legacy code and shouldn't be used in new code.

Reply:
Removed.